internal/api/client.go

```diff
@@ -1,15 +1,31 @@
 package api
 
-import "github.com/google/go-github/github"
+import (
+	"context"
+	"os"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
 
 // Client describe an instance of API client.
 type Client struct {
 	github *github.Client
 }
 
 // NewClient return a instance of github client.
-func NewClient() *Client {
+func NewClient(ctx context.Context) *Client {
+	token := os.Getenv("GITHUB_API_TOKEN")
```

Comment:
This code checks the environment variables to get the github token, simply.
But we need to use keychain which will implement by #5